test/fixtures/result_expected.js

```diff
@@ -15,7 +15,7 @@
       color: red;
     }
     smile:before {
-      content: "\000B";
+      content: "\\000B";
```

Comment:
Actually, because `\000B` appears inside a template literal in this file, the JS engine would simply throw an error for not allowing the deprecated octal literals.

It's accepted in non-strict mode ES5 though as `[null character]B`